### ISSUE TITLE
I can't submit the new version to Cocoapods

### ISSUE THREAD

**Issue by lucaszischka** [lucaszischka/BottomSheet#85]
# For some reason cocoa pods won't let me submit the new version.
Maybe one of you can help me. Otherwise I will need to drop cocoapods.

## The first issue
When trying to update the pod to version 3.0.0, I get the following error:
```
Cocoapod spec: Unable to find other source ref for `Contents.json`
```
But after short [research on StackOverflow](https://stackoverflow.com/questions/63120229/cocoapod-spec-unable-to-find-other-source-ref-for-contents-json) I found out that is probably caused by the new Image Asset I added. 
So I got working and after a (sadly not so) short time I got it error to disappear. Here is what I changed in the Podfile to make this error disappear:
```ruby
# Optional
spec.screenshots  = 'https://user-images.githubusercontent.com/63545066/132514316-c0d723c6-37fc-4104-b04c-6cf7bbcb0899.gif', 'https://user-images.githubusercontent.com/63545066/132514347-57c5397b-ec03-4716-8e01-4e693082e844.gif', 'https://user-images.githubusercontent.com/63545066/132514283-b14b2977-c5d1-4b49-96b1-19995cd5a41f.gif' # Added
spec.osx.deployment_target = '10.15' # Added
spec.preserve_paths = 'Sources/BottomSheet/**/*.xcassets' # Added
# Fixed the issue
spec.source_files = 'Sources/BottomSheet/**/*.swift' # was before: s.source_files = 'Sources/BottomSheet/**/*'
spec.resources = 'Sources/BottomSheet/**/*.xcassets' # Added
```
The first 3 are purely optional, but the last two lines fixed it. However a new error appeared. I included the first error only for context and as reference to explain the changes to the Podfile (**they are NOT committed jet**).

## The second issue
After getting the Podfile to resolve, cocoapods started to build the project to make sure that everything is working; however for some reason it isn't and the build always fails:
```
 -> BottomSheetSwiftUI (3.0.0)
    - ERROR | xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Using codesigning identity override: -
    - NOTE  | xcodebuild:  note: Build preparation complete
    - NOTE  | xcodebuild:  note: Planning
    - NOTE  | xcodebuild:  note: Building targets in dependency order
    - ERROR | xcodebuild:  BottomSheetSwiftUI/Sources/BottomSheet/BottomSheetView/BottomSheetView+HelperViews.swift:322:32: error: type 'Bundle' has no member 'module'
    - NOTE  | xcodebuild:  note: Using codesigning identity override: 

Analyzed 1 podspec.

[!] The spec did not pass validation, due to 2 errors.
```

The strange part is, that the project (obviously) builds fine. However I read [here on StackOverflow](https://stackoverflow.com/questions/31065447/no-such-module-when-i-use-cocoapods) that sometimes you need to change something in the build schemes of the project, to get this working.
However cocoapods handels it completely itself and I don't know how I could inject this build schemes change.
But honestly maybe even something else is the issue and it has nothing to do with build schemes.

## Final words
I spend my last 2 hours on searching for solutions, but without any luck. I don't have any ideas what I could try and also im never using cocoapods myself so I don't even have any experience.
Maybe this is a common issue and I just googled the wrong questions, so please help me out. But as I said earlier, if no-one can help me here I won't be able to publish a new cocoapods release.

**Comment by lucaszischka** [comment]
I think I found the solution thanks to this article:
https://medium.com/clutter-engineering/supporting-both-swift-package-manager-and-cocoapods-in-your-library-861f00b6b0f9
and this forum post
https://developer.apple.com/forums/thread/652736

**Comment by lucaszischka** [comment]
Fixed by dbfd88fe4b59cf570c0ebb635704231166ff617a
